addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/handler/MilightBridgeV6Handler.java

```diff
@@ -90,7 +90,12 @@ protected Runnable getKeepAliveRunnable() {
             @Override
             public void run() {
                 // logger.debug("Session keep alive");
-                session.keep_alive(refrehIntervalSec * 1000);
+                try {
+                    session.keep_alive(refrehIntervalSec * 1000);
```

Comment:
It would be great if `keepAlive` and the other method could follow the naming conventions. I don't know whether the other big PR is already merged, if that is not the case I would rather not do the renaming now because it will increase the chances of conflicts. But if you could plan to it in the near future that would be great.